src/base/bittorrent/sessionstatus.h

```diff
@@ -29,50 +29,43 @@
 #ifndef BITTORRENT_SESSIONSTATUS_H
 #define BITTORRENT_SESSIONSTATUS_H
 
-#include <libtorrent/session_status.hpp>
 #include <QtGlobal>
 
 namespace BitTorrent
 {
-    class SessionStatus
+    struct SessionStatus
     {
-    public:
-        SessionStatus(const libtorrent::session_status &nativeStatus);
```

Comment:
Now each branch is in their own function, one accepting alert, second use status object...
I don't want extract this code in case of libtorrent-1.1 since it will lead to create constructor that constructs object from very common data (session_stats_alert and struct of counter indicies).